searxng/searxng
# [fix] mobile result scaling in simple theme

## What does this PR do?

Fixes the issue where result page would scale weird on small width screens, because of some min width on the `input#q` element on the search form. This is achieved by setting the width of that element to 100% (basically 100% of the 1fr of the grid element so it scales correctly now, when the width decreases). Also this PR hides the watermark on device width smaller 20rem (ultra-small-phone) to make space for the search input field.
]The changes viewport basically sets the initial scale to 1 (so on high dpi screens the browser can zoom in by default. We should not change this since the browser provider probably knows best which zoom level to use  by default). The max-scale and user-scale got dropped  so the user can zoom in as much as they like which should help with accessibility...

[EDIT] thanks to https://github.com/ZetaTom for the fix! :D

## Why is this change important?

Make the simple theme A LOT more polished on mobile.

## How to test this PR locally?

`make run`

* result page for each category should scale correctly on mobile now
* there is still the issue with the image detail modal scaling incorrectly (https://github.com/searxng/searxng/issues/445). It seems that the image is way to big for the side in portrait with some sort of min-width now. But since it was broken before, just differently, I think its fine to merge this anyway and fix the image in another PR

## Author's checklist

Small Phone:
![image](https://user-images.githubusercontent.com/38733246/141368956-4027c1c5-de29-456d-a872-9f4510fcd2f1.png)

Ultra Small Phone:
![image](https://user-images.githubusercontent.com/38733246/141368994-2fed83dd-4a6a-4092-9644-7e36c91516c4.png)


## Related issues
